articles/active-directory/saas-apps/lessonly-tutorial.md

```diff
@@ -200,30 +200,30 @@ The objective of this section is to create a test user in the Azure portal calle
 
     d. Click **Create**.
  
-### Creating a Lesson.ly test user
+### Creating a Lessonly.com test user
 
-The objective of this section is to create a user called Britta Simon in Lesson.ly. Lesson.ly supports just-in-time provisioning, which is by default enabled.
+The objective of this section is to create a user called Britta Simon in Lessonly.com. Lessonly.com supports just-in-time provisioning, which is by default enabled.
 
-There is no action item for you in this section. A new user will be created during an attempt to access Lesson.ly if it doesn't exist yet.
+There is no action item for you in this section. A new user will be created during an attempt to access Lessonly.com if it doesn't exist yet.
 
 > [!NOTE]
-> If you need to create an user manually, you need to contact the [Lesson.ly support team](mailto:dev@lessonly.com).
+> If you need to create an user manually, you need to contact the [Lessonly.com support team](mailto:dev@lessonly.com).
 
 ### Assigning the Azure AD test user
 
-In this section, you enable Britta Simon to use Azure single sign-on by granting access to Lesson.ly.
+In this section, you enable Britta Simon to use Azure single sign-on by granting access to Lessonly.com.
 
 ![Assign User][200] 
 
-**To assign Britta Simon to Lesson.ly, perform the following steps:**
+**To assign Britta Simon to Lessonly.com, perform the following steps:**
 
 1. In the Azure portal, open the applications view, and then navigate to the directory view and go to **Enterprise applications** then click **All applications**.
 
 	![Assign User][201] 
 
-1. In the applications list, select **Lesson.ly**.
+1. In the applications list, select **Lessonly.com**.
 
-	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lesson.ly_app.png) 
+	![Configure Single Sign-On](./media/lessonly-tutorial/tutorial_lesson.ly_app.png)
```

Comment:
Same here @susanesho - can we revert this back to make sure the photo still displays properly?

Reply:
it is actually the same thing(the_same_url) except for white trailing space removed @case-eee